### PR TITLE
280 add mac install instructions

### PR DESCRIPTION
**Pull request by jsolbrig**
# Reviewer Checklist
https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#280

# Testing Instructions

- Check README.md links to Linux and Mac instructions
- Test installation on both Linux and Mac including tests in step 7.

# Summary

Installation Updates
====================

Add Mac installation instructions
---------------------------------

Added Mac installation instructions in mac_installation.rst and added link in the
readme. Also reworked order of Linux installation.

::

    modified: docs/source/starter/installation.rst
    created: docs/source/starter/mac_installation.rst
    modified: README.md

# Output

### CODE REVIEW

File: docs/source/starter/installation.rst [hunk 1]
```diff
@@ -114,7 +114,10 @@ This command installs all GeoIPS Python dependencies, and GeoIPS itself.
 .. code:: bash
 
     # Ensure geoips python environment enabled before installing geoips
-    pip install -e "$GEOIPS_PACKAGES_DIR/geoips"[doc,lint,test,debug]
+    # using "conda activate geoips"
+
+    # Install geoips via pip
+    pip install -e "$GEOIPS_PACKAGES_DIR/geoips[doc,lint,test,debug]"
```
Comment (mindyls) [inline docs/source/starter/installation.rst:120]: apparently this works on bash and zsh, the old way ONLY worked on bash.

**Review by mindyls — APPROVED** [review]
This just moved the git clone to AFTER the conda install / create (so the new git is available for the clone).  Works for me.

**Review by evrose54 — APPROVED** [review]
Tested out the install instructions and everything worked for me. Should be good to go!

File: docs/source/starter/installation.rst (hunk 1 above)
Comment (jsolbrig) [reply docs/source/starter/installation.rst:120]: Yeah, on zsh it thinks that the `[doc,lint,test,debug]` part is a part of the filename. So it looks for `$GEOIPS_PACKAGES_DIR/geoips[doc,lint,test,debug]`. Moving the quote sends the whole thing to pip, instead, and lets pip parse that info.

**Pull request merged**